### ISSUE TITLE
(Conclusions) Facebook comment is bit off

### ISSUE THREAD

**Issue by CharlesDLandau** [jjanenda/SQL_Scratch_Capstone_TurnIn_JJanenda#4]
The prompt asked for up to five campaigns, and Facebook has good conversion. I think there's a value proposition there which you aren't capturing in your analysis.

**Comment by jjanenda** [comment]
I must not have communicated this well enough.  The only campaign that used Facebook was called 'retargeting-ad'.  I recommended continuing that campaign.   From the ppt. 

'Further suggest extending retargeting ad campaigns as they have the potential to reach 100% of prospects that interact with the site, INCLUDING those that do not provide an email address.  Would strongly consider purchasing ads on additional sites ... as they generate excellent traffic from 'story' campaigns'.    

So I was trying to say that retargeting ads (of which Facebook was the only one) were GREAT because you didn't need an email address to reach prospects and, if possible, EXTEND them because Facebook was only 4th out the other sites at generating traffic.  

So, I did include Facebook as one of the five campaigns to continue.  Sorry if I didn't communicate that well.

Also, sorry for the OCD that makes me think I need to respond to these, even though you gave me full credit for the answer.  ;-)
